shared/src/main/kotlin/com/egm/stellio/shared/model/NgsiLdEntity.kt

```diff
@@ -195,6 +195,8 @@ class NgsiLdPropertyInstance private constructor(
             val attributes = getNonCoreAttributes(values, NGSILD_PROPERTIES_CORE_MEMBERS)
             val relationships = getAttributesOfType<NgsiLdRelationship>(attributes, NGSILD_RELATIONSHIP_TYPE)
             val properties = getAttributesOfType<NgsiLdProperty>(attributes, NGSILD_PROPERTY_TYPE)
+            if (attributes.size > relationships.size + properties.size)
```

Comment:
The same check should not be added to NgsiLdEntity also ?

Reply:
Hmm, it is done (L.154)

Reply:
OK for this one?

Reply:
Yes, i''ll resolve it